test/TimeAdapterTestUtilities/conanfile.py

```diff
@@ -11,22 +11,10 @@ class TimeAdapterTestUtilitiesConan(ConanFile):
     license = "MIT"
     generators = "cmake_find_package"
     settings = "os", "compiler", "build_type", "arch"
-    options = {"gtest": ["1.7.0", "1.8.1","1.10.0"]}
-    default_options = {"gtest":"1.10.0"}
     exports_sources = "*", "!build-*"
```

Comment:
Change export_sources "!build-*" to "!build*"
On multiple instances of conan install it will delete all previous packages due to the new files being generated and added as export_sources. You are generating new build directories inside build/ and not on diferent build-*/

Reply:
Fixed